packages/catalog-realm/SkillCard/search-replace-editing.json

```diff
@@ -0,0 +1,18 @@
+      "commands": [],
+      "title": "Search/replace code editor",
```

Comment:
Do you think we can come up with a better title so it's not too confusing for the user?

<img width="351" alt="image" src="https://github.com/user-attachments/assets/4b75a298-8248-45b4-9d96-31252dbe68ed" />

Perhaps this should just be "Code editing", and the other skill that has instructions for gts file can be "Card definition editing"?

Reply:
"Source code editing"

Reply:
"Boxel Coding"

Reply:
Also should this skill be in base realm instead?

Reply:
I have this commit in my work in progress PR that changes skill names as suggested, and adds them as default in code mode: https://github.com/cardstack/boxel/pull/2245/commits/0da24a98871378eec5da3757d89badbaefdee0aa

@IanCal If you like we can cherry pick it and add to this PR